lib/api/controllers/adminController.js

```diff
@@ -64,6 +88,28 @@ function AdminController(kuzzle) {
   };
 
   /**
+   * Get the user mapping
+   * @param {RequestObject} requestObject
+   * @param {Object} userContext
+   * @returns {Promise}
+   */
+  this.getUserMapping = function adminGetUserMapping (requestObject, userContext) {
+    var
+      modifiedData = null;
+
+    return kuzzle.pluginsManager.trigger('data:beforeGetUserMapping', {requestObject, userContext})
+      .then(data => {
+        modifiedData = data;
```

Comment:
To build the response object later on.
But this code will eventually have to be entirely rewritten following #559